packages/flutter/lib/src/material/text_selection_toolbar.dart

```diff
@@ -539,10 +540,8 @@ class _RenderTextSelectionToolbarItemsLayout extends RenderBox with ContainerRen
       } else {
         childParentData.offset = Offset(0.0, overflowHeight);
         overflowHeight += child.size.height;
-        nextSize = Size(
-          math.max(child.size.width, nextSize.width),
-          overflowHeight,
-        );
+        overflowWidth = math.max(child.size.width, nextSize.width);
+        nextSize = Size(overflowWidth, overflowHeight);
```

Comment:
Is the introduction of overflowWidth just an attempt to make this easier to read, or am I misreading this? I think you can declare it here rather than up above, right?

Reply:
Oh good catch: I first introduced it because I need it at the end of this method and after moving the logic to a separate method I kept it for readability. I will move the declaration as suggested, thanks.

Reply:
Just push an update and finally I removed this variable as it does not improve readability.